chapters/cmdbuffers.txt

```diff
@@ -761,14 +761,6 @@ time.
 include::{generated}/validity/structs/VkCommandBufferInheritanceInfo.txt[]
 --
 
-If ename:VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT was not set when
-creating a command buffer, that command buffer must: not be submitted to a
-queue whilst it is already in the <<commandbuffers-lifecycle, pending
-state>>.
-If ename:VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT is not set on a
-secondary command buffer, that command buffer must: not be used more than
-once in a given primary command buffer.
-
```

Comment:
I don't think we should remove this - there's some plan to change/move instances of this type of language fairly globally at some point; but they were added for a reason. This is almost always as a highlight for non-obvious behavior which we expect developers to trip up on; I would be ok changing these to notes though.

Reply:
I dunno, feels obvious because this just reiterates what `VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT` is defined as.

Reply:
Ah yes, looking at it I think you're right - so I'm fine with accepting this.

Reply:
I already preemptively made it a note. But I cannot pass on deleting stuff, which is what I think you are agreeing with here. So I will do that when I get to PC